test/functional/organizations.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+require('../testHelpers');
+const _ = require('lodash');
+
+exports.lab = Lab.script();
+
+const server = Server.server;
+
+let Organization;
+
+describe('Organizations resources', () => {
+  let firstOrganization, secondOrganization;
+  let superUser, userWithMultipleRoles, userWithNoRoles, userWithOneRole;
+
+  before(async () => {
+    await Server.initialize();
+    const setup = await createTestSetup();
+    ({Organization} = server.models);
+    [userWithOneRole, superUser, userWithMultipleRoles, userWithNoRoles] = setup.users;
+    [firstOrganization, secondOrganization] = setup.organizations;
+  });
+
+  after(() => {
+    return resetTestSetup();
+  });
+
+  describe('GET /permissions', () => {
+    let multiRoleToken, noRoleToken, regularToken, superToken;
+
+    before(async () => {
+      [superToken, regularToken, multiRoleToken, noRoleToken] = await Promise.all([
+        logIn(superUser),
+        logIn(userWithOneRole),
+        logIn(userWithMultipleRoles),
+        logIn(userWithNoRoles)
+      ]);
+    });
+
+    function callServer(token, test) {
+      return runTest({
+        method: 'GET',
+        url: '/organizations',
+        headers: {
+          'Authorization': token,
+          'Device': 'client'
+        }
+      }, test);
+    }
+
```

Comment:
below: how can i control this? I allow the superUsers (with superToken) list organizations in the handler, but i got a Internal Error when i run the test.. idk please some help
Also, how can i control the error 401 (if not logged) and 403 (if not a superUser and cant see all organizations)